self-tests.yml

```diff
@@ -800,3 +827,11 @@ buildvariants:
         distros:
           *generic-linux-distros
       - name: ".xc"
+
+  - name: js-tests
```

Comment:
Since you said they're broken, should we avoid running the tests for now?

Reply:
I agree. Maybe we should disable all test except the ones that pass? Or except a single one that noops?